src/test/java/teammates/test/cases/ui/browsertests/AllAccessControlUiTests.java

```diff
@@ -157,7 +158,7 @@ public void testPubliclyAccessiblePages() throws Exception {
         ______TS("unauthorized page");
         AppUrl url = createUrl(Const.ViewURIs.UNAUTHORIZED);
         currentPage.navigateTo(url);
-        verifyRedirectToNotAuthorized();
+        currentPage.verifyHtml("/unauthorized.html");
```

Comment:
I'd say it's redundant to do > 1 HTML check for the same page. Also, this is to make `testPubliclyAccessiblePages` look uniform.